dnsrecon/cli.py

```diff
@@ -385,8 +385,8 @@ def brute_srv(res, domain, verbose=False, thread_num=None):
             future_results = {executor.submit(res.get_srv, srvtype + domain): srvtype for srvtype in srvrcd}
             # Display logs as soon as a thread is finished
             for future in futures.as_completed(future_results):
-                res = future.result()
-                for type_, name_, target_, addr_, port_, priority_ in res:
+                result = future.result()
```

Comment:
If I understand the code correctly, this was variable shadowing and not intentional.